### ISSUE TITLE
Import Rules in Lulu 2.0.0

### ISSUE THREAD

**Issue by cesmed** [objective-see/LuLu#265]
Hi,
  I have a simple question: 

  Now, in a new version 2.0.0 ¿how can I import my old rules? I saved them in a file .plist but I don't know how can I import in a new version...

Thanks.

**Comment by Sellicks** [comment]
Hi,
I have the same question too.
New Lulu doesn't have import/export option anymore. Is there a way around?
macOS Catalina.
Thanks in advance!

**Comment by bronson** [comment]
Agreed. Plus it would be nice to be able to edit rules with a text editor.

**Comment by justinormont** [comment]
As a workaround, the rules are kept in in the OS (not user) level library folder: `/Library/Objective-See/LuLu/rules.plist`. 

I'm unsure if saving/restoring this file is sufficient though. A proper export/import will be more stable if for instance the plist contents changes.

Related/duplicate issues:
* https://github.com/objective-see/LuLu/issues/364
* https://github.com/objective-see/LuLu/issues/334
* https://github.com/objective-see/LuLu/issues/439
* https://github.com/objective-see/LuLu/issues/2 (original import/export issue)

**Comment by SawkeeReemo** [comment]
Just chiming in here to ask for the same thing! I'm running the latest version 2.4.2, and have been thinking about rebuilding this system. Don't want to have to rebuild all these rules! Can anyone confirm if this plist trick works?
